### PR TITLE
Sitemaps guide

### PR DESCRIPTION
**Pull request by blakewilson**
## Description

This PR introduces a how-to guide for creating sitemaps with Faust.js

## Documentation Changes

* Adds a new guide called "Sitemaps"
* Adds a new sidebar path for "Sitemaps" under Guide

### CODE REVIEW

File: internal/website/src/components/MetaTags.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```
Comment (theodesp) [inline internal/website/src/components/MetaTags.js:1]: You will probably want to do a git merge git push here to remove those merged changes

File: internal/website/docs/next/guides/sitemaps.mdx [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    },
+  });
+
+  if (sitemapRequest) {
+    return sitemapRequest;
+  }
+
+  return NextResponse.next();
+}
+```
```
Comment (josephfusco) [inline internal/website/docs/next/guides/sitemaps.mdx:232]: Note that `/sitemap.xml` redirects to `/wp-sitemap.xml` or if Yoast is activated, `/sitemap_index.html`. Knowing that would we still need to include both sitemap examples?

File: internal/website/src/components/MetaTags.js (hunk 1 above)
Comment (blakewilson) [reply internal/website/src/components/MetaTags.js]: Oops! Fixed.

File: internal/website/docs/next/guides/sitemaps.mdx (hunk 2 above)
Comment (blakewilson) [reply internal/website/docs/next/guides/sitemaps.mdx:232]: @josephfusco That was not my experience when using the default WordPress sitemap. For me, it only exposed `/wp-sitemap.xml`. Could it have been that you experienced this when permalinks were not refreshed after disabling Yoast?

https://make.wordpress.org/core/2020/07/22/new-xml-sitemaps-functionality-in-wordpress-5-5/

Comment (josephfusco) [reply internal/website/docs/next/guides/sitemaps.mdx:232]: Here are my findings with this.

WP install with no plugins,
✅ **/sitemap.xml** successfully redirects to **/wp-sitemap.xml**

If I activate Yoast, & do not flush WP permalinks,
❌ **/sitemap.xml** redirects but shows 404.

If I activate Yoast, & flush WP permalinks,
✅ **/sitemap.xml** successfully redirects to **/sitemap_index.html**

If I deactivate Yoast, & do not flush WP permalinks,
❌ **/sitemap.xml** redirects but shows 404.

If I deactivate Yoast & flush WP permalinks,
✅ **/sitemap.xml** successfully redirects to **/wp-sitemap.xml**

Giving this more thought, I'm good with us being explicit here as I'm not a fan of asking a developer to flush permalinks.

**Pull request merged**